ESSArch_Core/configuration/views.py

```diff
@@ -106,6 +138,7 @@ class SysInfoView(APIView):
     """
 
     def get(self, request):
+        full = str_2_bool_convert(request.query_params.get('full', False))
```

Comment:
Use `string_to_bool` from `filters.py`
```suggestion
        full = string_to_bool(request.query_params.get('full', 'false'))
```